views/account/generate_qr_code/main.js

```diff
@@ -0,0 +1,27 @@
+let qrcode_div = document.getElementById("qrcode-div");
+let qrcode_submit = document.getElementById("qr_submit");
+let url = document.getElementById('url');
+let size = document.getElementById('size');
+
+// Button submit
+let onGenerateSubmit = async function (e) {
+  e.preventDefault();
+
+  // Clear the div's contents
+  qrcode_div.innerHTML = "";
+
+  // Create the QR code using the google charts api
+  let qrcode = document.createElement("img");
```

Comment:
Thanks for changing to Google Leah. Better safe than sorry I think!

Reply:
And obviously, this function will need to change as we get further into the project. The user won't have an option to manually enter a link, obviously. Some of this code will likely need to be in an API endpoint on our server to process what accounts they want, and the server can reply with the correct link. But this is good in the client script file for showing her in our code review next week.

Reply:
Yeah no worries princess; always gotta keep you happy 😂 
Yes I agree, it will definitely change over the course of the project. I just had some of the stuff like inserting a manual link on the page as like a placeholder before we actually implement a link on our side.